pkg/cmd/roachtest/versionupgrade.go

```diff
@@ -503,7 +503,9 @@ func makeVersionFixtureAndFatal(
 			// TODO(irfansharif): This will need replacement, or a suitable
 			// alternative, for when we strip out rocksdb entirely. This will
 			// fail with an opaque error if building the cockroach binary
-			// without rocksdb (`make buildshort`).
+			// without rocksdb (`make buildshort`). Pebble already supports
+			// checkpointing (see CreateCheckpoint in pebble.go). This just needs
+			// to be exposed to a command line method.
```

Comment:
I think we'll need to do it for this one before merge. Every release requires us to generate fixtures for it, and we do it through this codepath.

Reply:
Ack. Probably worthwhile to add a `pebble checkpoint` tool. As Bilal mentioned, all the hard work is already done.

Reply:
This is resolved now that #55751 is in. I've also un-skipped the roachtest.